Common/Kernel/CustomExceptionHandler.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.Threading.Tasks;
+using DTO;
+using Microsoft.AspNetCore.Diagnostics;
+using System.Text.Json;
+using Kernel.CustomExceptions;
+
+namespace Kernel
+{
+    public static class CustomExceptionHandler
+    {
+        public static async Task HandleCustomException(HttpContext context)
```

Comment:
Rename in Handle(). It is already static class CustomExceptionHandler and there is some duplications. CustomExceptionHandler.HandleCustomException()